app/views/admin/creator_settings/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if forem_creator_flow_enabled? && User.with_role(:creator).present? %>
```

Comment:
@juliannatetreault I think this may be why the form is not rendering :) It should have been `FeatureFlag.enabled?(:creator_onboarding)` instead of `forem_creator_flow_enabled? `.